src/main/java/uk/gov/hmcts/reform/cwrdapi/service/ValidationService.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.cwrdapi.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.cwrdapi.client.domain.CasWorkerDomain;
+import uk.gov.hmcts.reform.cwrdapi.domain.ExceptionCaseWorker;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import javax.validation.ConstraintViolation;
+
```

Comment:
can you please  create interface and implementation classes in  service package

Reply:
Yes sure.